### PR TITLE
added get directions, applied getting of address from mapbox to match…

### PR DESCRIPTION
**Pull request by RazielGelo**
- added get directions capability
- updated location to display actual address instead of fake data

### CODE REVIEW

File: src/pages/match/[id]/view.tsx [hunk 1]
```diff
@@ -1,68 +1,182 @@
 //third-party import
-import { getMatchById } from "@/lib/actions/match";
-import Database from "@/lib/resources/database";
-import { Match } from "@/lib/types/Match";
```
Comment (dmncrynn) [inline src/pages/match/[id]/view.tsx:4]: things should be in local import:

- Database
- Match

File: src/pages/match/[id]/view.tsx [hunk 2]
```diff
@@ -1,68 +1,182 @@
...
     }
 
-    return(
+    // Function to handle get direction click event
+    async function getDirectionsClicked() {
+
+        // fetch mapbox api using directions services
+        try {
+            const endpoint = `https://api.mapbox.com/directions/v5/mapbox/driving/${
+        startLocation!.lng
```
Comment (dmncrynn) [inline src/pages/match/[id]/view.tsx:133]: add comments

File: src/pages/match/[id]/view.tsx [hunk 3]
```diff
@@ -126,28 +256,27 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
         const match = await getMatchById(id as string);
```
Comment (dmncrynn) [inline src/pages/match/[id]/view.tsx:256]: add comments

**Review by dmncrynn — CHANGES_REQUESTED** [review]
Please review comments added! before getting approved for merging

File: src/pages/match/[id]/view.tsx (hunk 1 above)
Comment (RazielGelo) [reply src/pages/match/[id]/view.tsx:4]: This is deleted, already moved and corrected the imports

File: src/pages/match/[id]/view.tsx (hunk 2 above)
Comment (RazielGelo) [reply src/pages/match/[id]/view.tsx:133]: added comments

File: src/pages/match/[id]/view.tsx (hunk 3 above)
Comment (RazielGelo) [reply src/pages/match/[id]/view.tsx:256]: added comments

**Review by dmncrynn — APPROVED** [review]

**Pull request merged**